wasp/apps/alarm.py

```diff
@@ -0,0 +1,155 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2020 Daniel Thompson
+# Copyright (C) 2020 Joris Warmbier
+"""Alarm Application
+~~~~~~~~~~~~~~~~~~~~
+
+An application to set a vibration alarm. All settings can be accessed from the Watch UI
+"""
+
+import wasp
+import icons
+import time
+
+icon = (
```

Comment:
Please keep the comment giving the original file name and size (and also add the original png file to the res/ directory to make future changes easier).